src/lighteval/metrics/metrics_sample.py

```diff
@@ -926,7 +926,7 @@ def compute(self, sample_ids: list[str], responses: list, formatted_docs: list[D
         """
         questions = [formatted_doc.specific["question"] for formatted_doc in formatted_docs]
         options = [formatted_doc.choices for formatted_doc in formatted_docs]
-        golds = [formatted_doc.choices[formatted_doc.gold_index[0]] for formatted_doc in formatted_docs]
+        golds = [formatted_doc.get_golds() for formatted_doc in formatted_docs]
```

Comment:
cc @NathanHB (is there a reason for not using the get_golds function?)

Reply:
No just an error, my bad !

Reply:
this needs to be an str and not a list though. I guess we could just take the first element

Reply:
The first element of get_golds()?

Reply:
yes ! Or, if you need all the available golds, we could change the signature of the judge prompt templates to take a list for the gold answers.

Reply:
For me one is enough. I'll change it accordingly.